pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -404,6 +408,18 @@
   front_join_us_challenge: "Win a video chat for your classroom with a celebrity!"
   front_join_us_learn_more: "Learn more"
 
+  cta_see_whats_new: "See what's new"
+  cta_try_more_activities: 'Try more activities'
+  cta_try_an_activity: 'Try an activity'
+
+  hoc_homepage_join_the_movement_heading: 'Join the movement'
+  hoc_homepage_join_the_movement_description: 'Hour of Code is available year round, but every year in December your class can join millions of students around the world celebrating Computer Science Education Week with the Hour of Code. Registration for the annual celebration starts each year in October.'
+  hoc_homepage_what_is_hoc_subhead: 'One-hour tutorials in over 45 languages. No experience needed. Hour of Code activities are available for free year-round.'
+  hoc_homepage_what_is_hoc_description: 'The Hour of Code is a one-hour introduction to computer science, using fun tutorials to show that anybody can learn the basics. This grassroots campaign is supported by over 400 partners and 200,000 educators worldwide.'
+  hoc_homepage_what_is_hoc_cta_how_to_guide: "or <a href='%{howto_guide_url}'>read our How-to guide</a> to plan an event for your class"
```

Comment:
Where is the first part of this sentence?  

We generally want to translate entire sentences at a time, in case another language can't rely on this particular ordering.

Reply:
In addition, I think we generally try to use markdown instead of `a` tags. So "[read our How-to guide]({% howto_guide_url})" in this case

Reply:
The first part of the sentence is a button that says "Try an activity" (`cta_try_an_activity: 'Try an activity'`) — I'll remove the "or" and make it "[Read our How-to guide]({% howto_guide_url}) to plan an event for your class."

Here it is in context w/o the "or":
<img width="450" alt="Screen Shot 2022-09-20 at 12 12 41 PM" src="https://user-images.githubusercontent.com/9256643/191344410-bdc5ce4b-7a54-4f98-b883-db8c44dfa5e5.png">